### PR TITLE
chore: fixes for example configuration files

### PR DESCRIPTION
**Pull request by neilconway**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Review by naren-determined — CHANGES_REQUESTED** [review]
For `cifar10_pytorch` and `cifar10_tf_keras` since they are both based off of the same example and looking at the original code here (https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn.py), seems like we should only be running for 25 or 50 epochs (not 32) and reporting the same accuracy metrics mentioned here. Everything else looks good!

**Comment by neilconway** [comment]
> For cifar10_pytorch and cifar10_tf_keras since they are both based off of the same example and looking at the original code here (https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn.py), seems like we should only be running for 25 or 50 epochs (not 32) and reporting the same accuracy metrics mentioned here.

Running this for 32 epochs does get us close to around 74% accuracy:

http://ec2-44-236-60-76.us-west-2.compute.amazonaws.com:8080/det/trials/203 (tf keras)
http://ec2-44-236-60-76.us-west-2.compute.amazonaws.com:8080/det/trials/204 (pytorch)

I think we'd be a bit lower if we trained for only 25; not sure how accurate we'd get after 50. But I think training for 32 gets us to about the metrics we mention.

**Review by naren-determined — APPROVED** [review]
Ok, in the case feel free to merge!
